app/javascript/tests/patient/monitoring_actions/CaseStatus.test.js

```diff
@@ -134,7 +134,8 @@ describe('CaseStatus', () => {
     expect(wrapper.state('case_status')).toEqual('Suspect');
     expect(wrapper.state('confirmedOrProbable')).toBe(false);
     expect(wrapper.state('isolation')).toBe(false);
-    expect(modalBody.find('p').text()).toEqual('Are you sure you want to change case status to "Suspect"? The monitoree will be placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate to continue exposure monitoring and the Latest Public Health Action will be set to "None".');
+    expect(modalBody.find('p').text()).toContain('Are you sure you want to change case status to "Suspect"? The monitoree will be placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate to continue exposure monitoring.');
```

Comment:
can these be combined to a `toEqual`?

Reply:
@sgober and I messaged about this a bit and tested out what the `toEqual` check would look like and we both agree in the end that splitting it up between 2 `toContain` statements was the cleanest approach.